netlify.toml

```
@@ -23,4 +23,4 @@
 
 [functions]
   directory = "netlify/functions/"
-  included_files = ["_data/contests/*", "_data/handles/*.json"]
+  included_files = ["_data/contests/*", "_data/findings/*", "_data/handles/*.json", "_data/handles/avatars/*"]
```

```suggestion
  included_files = ["_data/contests/*", "_data/findings/*", "_data/handles/*.json"]
```

I added this for testing purposes, but as we grabbed the avatar differently, we can revert back here :)

---

Thanks for point this out with a patch!